src/plugins/gui/scenes/AddressFormScene.tsx

```diff
@@ -249,73 +249,81 @@ export const AddressFormScene = React.memo((props: Props) => {
-        <GuiFormField
-          fieldType="address"
-          autofocus
-          label={lstrings.form_field_title_address_line_1}
-          value={formData.address}
-          fieldRef={rAddressInput}
-          onChangeText={handleChangeAddress}
-          onFocus={handleShowAddressHints}
-          onBlur={handleHideAddressHints}
-        />
-        <Animated.View style={[Platform.OS === 'ios' ? styles.dropContainer : styles.dropContainerAndroid, aDropContainerStyle]}>
-          <ScrollView keyboardShouldPersistTaps="always" nestedScrollEnabled>
-            {searchResults.map(searchResult => {
-              const displaySearchResult = `${searchResult.address}\n${searchResult.city}, ${searchResult.state}, ${countryCode}`
-              return (
-                <TouchableOpacity key={searchResults.indexOf(searchResult)} onPress={addressHintPress(searchResult)}>
-                  <View style={styles.rowContainer} onLayout={handleHintLayout}>
-                    <EdgeText style={styles.addressHintText} numberOfLines={2}>
-                      {displaySearchResult}
-                    </EdgeText>
-                  </View>
-                </TouchableOpacity>
-              )
-            })}
-          </ScrollView>
-        </Animated.View>
-        <GuiFormField
-          fieldType="address2"
-          label={lstrings.form_field_title_address_line_2}
-          value={formData.address2}
-          onChangeText={handleChangeAddress2}
-          onBlur={handleHideAddressHints}
-        />
-        <GuiFormField
-          fieldType="city"
-          label={lstrings.form_field_title_address_city}
-          value={formData.city}
-          onChangeText={handleChangeCity}
-          onBlur={handleHideAddressHints}
-        />
-        <GuiFormField
-          fieldType="state"
-          label={lstrings.form_field_title_address_state_province_region}
-          value={formData.state}
-          onChangeText={handleChangeState}
-          onBlur={handleHideAddressHints}
-        />
-        <GuiFormField
-          fieldType="postalcode"
-          returnKeyType="done"
-          label={lstrings.form_field_title_address_zip_postal_code}
-          value={formData.postalCode}
-          onChangeText={handleChangePostalCode}
-          onBlur={handleHideAddressHints}
-        />
-        <MainButton
-          label={lstrings.string_next_capitalized}
-          marginRem={[1, 0.5, 1, 0.5]}
-          type="secondary"
-          disabled={disableNextButton}
-          onPress={handleSubmit}
-        />
-      </KeyboardAwareScrollView>
+    <SceneWrapper background="theme" hasNotifications hasOverscroll={false}>
+      {gap => (
+        <KeyboardAwareScrollView
+          keyboardShouldPersistTaps="handled"
+          extraScrollHeight={theme.rem(2.75)}
+          enableAutomaticScroll
+          enableOnAndroid
+          contentContainerStyle={{ paddingBottom: gap.bottom }}
```

Comment:
`style={ marginBottom: -gap.bottom }`

Reply:
No this is right, it should be positive. The notificationHeight is meant as an
contentContainerStyle paddingBottom (overscroll) - using it as a style padding causes a visual cutoff